package.json

```diff
@@ -35,10 +36,13 @@
     "eslint-plugin-react": "7.16.0",
     "npm-run-all": "4.1.5",
     "prettier": "1.19.1",
-    "prop-types": "15.7.2"
+    "prop-types": "15.7.2",
+    "cpx": "1.5.0",
```

Comment:
Can we use cpx2 please? That seems to be the supported version now based on https://github.com/mysticatea/cpx/pull/49 (there's also a minor security issue in cpx according to `npm audit`.

Reply:
Good thinking. I didn't know `cpx2` was a thing!